### DISCUSSION TITLE
Scale makers on zoom

### DISCUSSION THREAD

**Discussion by mtnptrsn** [rnmapbox/maps#1338] [Q&A] [answered]
Hi!

I need to scale markers as you zoom, in order to show the true size of the marker on the map. here's a demonstration on what I want to achieve: https://codepen.io/mtnptrsn/pen/vYgodEb?editors=1001

On the web SDK, this is easy, but is this possible to achieve with react-native-mapbox-gl? If so, how?

Thanks.

**Comment by ferdicus** [comment, marked as answer]
You can do all sorts of things with expressions within the styles:
https://github.com/react-native-mapbox-gl/maps/blob/master/docs/StyleSheet.md

Check out the `example` folder for some inspiration.

**Reply by mtnptrsn** [reply]
Exactly what I was looking for. Thanks a lot for pointing me in the right direction! :)

**Reply by ferdicus** [reply]
you're welcome
